wire/msgtx_test.go

```diff
@@ -418,13 +504,22 @@ func TestTxSerialize(t *testing.T) {
 			multiTxPkScriptLocs,
 			false,
 		},
+		// Multiple outputs witness transaction.
+		{
+			multiWitnessTx,
+			multiWitnessTx,
+			multiWitnessTxEncoded,
+			multiWitnessTxPkScriptLocs,
+			true,
+		},
 	}
 
 	t.Logf("Running %d tests", len(tests))
 	for i, test := range tests {
 		// Serialize the transaction.
 		var buf bytes.Buffer
-		err := test.in.Serialize(&buf)
+		var err error
```

Comment:
Hmm, not quite sure what I was thinking in this case. Fixed to not declare the error up front.